db/migrate/20180119200528_add_subsample_to_sample.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubsampleToSample < ActiveRecord::Migration[5.1]
+  def change
+    add_column :samples, :subsample, :integer
```

Comment:
what sql data type is integer?   I hope it can hold the number 1,000,000.

Reply:
Maximum value is 2,147,483,647 according to http://sqlserverplanet.com/tsql/sql-server-max-int-value